spec.html

```diff
@@ -2184,42 +2184,51 @@ <h1>
-            1. If _k_ &le; _n_ &le; 21, return the string-concatenation of:
-              * the code units of the _k_ digits of the decimal representation of _s_ (in order, with no leading zeroes)
-              * _n_ - _k_ occurrences of the code unit 0x0030 (DIGIT ZERO)
-            1. If 0 &lt; _n_ &le; 21, return the string-concatenation of:
-              * the code units of the most significant _n_ digits of the decimal representation of _s_
-              * the code unit 0x002E (FULL STOP)
-              * the code units of the remaining _k_ - _n_ digits of the decimal representation of _s_
-            1. If -6 &lt; _n_ &le; 0, return the string-concatenation of:
-              * the code unit 0x0030 (DIGIT ZERO)
-              * the code unit 0x002E (FULL STOP)
-              * -_n_ occurrences of the code unit 0x0030 (DIGIT ZERO)
-              * the code units of the _k_ digits of the decimal representation of _s_
-            1. Otherwise, if _k_ = 1, return the string-concatenation of:
+            1. [id="step-number-tostring-intermediate-values"] Otherwise, let _n_, _k_, and _s_ be integers such that _k_ &ge; 1, _radix_<sup>_k_ - 1</sup> &le; _s_ &lt; _radix_<sup>_k_</sup>, 𝔽(_s_ &times; _radix_<sup>_n_ - _k_</sup>) is _x_, and _k_ is as small as possible. Note that _k_ is the number of digits in the representation of _s_ using using radix _radix_, that _s_ is not divisible by _radix_, and that the least significant digit of _s_ is not necessarily uniquely determined by these criteria.
+            1. If _k_ &le; _n_, then
+              1. If _radix_ ≠ 10 or _n_ &le; 21, return the string-concatenation of:
+                * the code units of the _k_ digits of the representation of _s_ using radix _radix_
+                * _n_ - _k_ occurrences of the code unit 0x0030 (DIGIT ZERO)
+            1. If _n_ &gt; 0, then
+              1. If _radix_ ≠ 10 or _n_ &le; 21, return the string-concatenation of:
+                * the code units of the most significant _n_ digits of the representation of _s_ using radix _radix_
+                * the code unit 0x002E (FULL STOP)
+                * the code units of the remaining _k_ - _n_ digits of the representation of _s_ using radix _radix_
+            1. If _n_ &le; 0, then
+              1. If _radix_ ≠ 10 or _n_ &gt; -6, return the string-concatenation of:
+                * the code unit 0x0030 (DIGIT ZERO)
+                * the code unit 0x002E (FULL STOP)
+                * -_n_ occurrences of the code unit 0x0030 (DIGIT ZERO)
+                * the code units of the _k_ digits of the representation of _s_ using radix _radix_
+            1. Assert: _radix_ is 10.
+            1. If _n_ &lt; 0, then
+              1. Let _sign_ be the code unit 0x002D (HYPHEN-MINUS).
+            1. Else,
+              1. Let _sign_ be the code unit 0x002B (PLUS SIGN).
```

Comment:
This alias could be more clearly named.
```suggestion
            1. If _n_ &lt; 0, then
              1. Let _exponentSign_ be the code unit 0x002D (HYPHEN-MINUS).
            1. Else,
              1. Let _exponentSign_ be the code unit 0x002B (PLUS SIGN).
```